src/cuda/gpu_cew_xc.h

```diff
@@ -0,0 +1,287 @@
+/*
+  !---------------------------------------------------------------------!
+  ! Written by Madu Manathunga on 09/29/2021                            !
+  !                                                                     ! 
+  ! Copyright (C) 2020-2021 Merz lab                                    !
+  ! Copyright (C) 2020-2021 Götz lab                                    !
+  !                                                                     !
+  ! This Source Code Form is subject to the terms of the Mozilla Public !
+  ! License, v. 2.0. If a copy of the MPL was not distributed with this !
+  ! file, You can obtain one at http://mozilla.org/MPL/2.0/.            !
+  !_____________________________________________________________________!
+
+  !---------------------------------------------------------------------!
+  ! This source file contains preprocessable functions required for     ! 
+  ! QUICK GPU version.                                                  !
+  !---------------------------------------------------------------------!
+*/
+
+#ifdef CEW
+#include "iface.hpp"
+#endif
```

Comment:
Why don't we preprocess away everything if we do not compile with CEw support? Or do not compile this file at all?

Reply:
Fixed.